### DISCUSSION TITLE
Why so many request for partial ? (206)

### DISCUSSION THREAD

**Discussion by MatthD** [katspaugh/wavesurfer.js#3878] [Q&A]
Hello,

I am using the React version of the wavesurfer `"@wavesurfer/react": "^1.0.7",`
I am checking the network tab & I don't understand why but it seems that when I am moving the cursor it's calling the url with a 206 partial response..

I don't understand the why of that because I am not using streaming and this make a lot of calls.

```jsx

      <div className="fancy-border" id="waveform">
        <WavesurferPlayer
          // backend="WebAudio"
          waveColor="rgb(200, 0, 200)"
          progressColor="rgb(100, 0, 100)"
          url={urlSigned}
          onReady={onReady}
          autoScroll={true}
          mediaControls={true}
          barHeight={0.8}
          dragToSeek={true}
          sampleRate={16000}
          plugins={plugins}
          minPxPerSec={20}
          splitChannels={overlay}
          height={500 / numberOfChannels}
          media={audioDomElement}
          peaks={peaks}
        />
      </div>
```

![Uploading Sep-23-2024 20-05-09.gif…]()

**Comment by katspaugh** [comment]
That’s how html audio works. It makes range requests for portions of audio as it plays.
